developer-workflow/c-api.rst

```diff
@@ -29,13 +31,18 @@ for building CPython itself, as indicated by a macro like ``Py_BUILD_CORE``.
 
 While internal API can be changed at any time, it's still good to keep it
 stable: other API or other CPython developers may depend on it.
+For users, internal API is sometimes the best a workaround for a thorny problem
+-- though those use cases should be discussed on `Discourse <https://discuss.python.org/c/30>`_
```

Comment:
```suggestion
--- though those use cases should be discussed on the
`C API Discourse category <https://discuss.python.org/c/30>`_
```

Both for a11y purposes and for all users, the link title should be appropriately accurate and descriptive—this is a link to the C API Discourse (itself currently a sub-category of the Core Dev category), not to the Discourse instance as a whole as the current text implies.

Also, since I changed the line anyway, fixed a en dash that should have been an em dash